src/UI/Utils/useFeatures.tsx

```diff
@@ -11,7 +11,7 @@ const extractFeatures = (fileContent) => {
 };
 
 const useFeatures = (): string[] => {
-  const [features, setFeatures] = useState([]);
+  const [features, setFeatures] = useState([""]);
```

Comment:
It was an empty array initially, maybe best to keep it that way.

Reply:
my bad